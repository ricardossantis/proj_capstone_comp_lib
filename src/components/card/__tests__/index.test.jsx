import React from 'react'
import Card from '../index'
import renderer from 'react-test-renderer'

it('renderer', () => {
  const tree = renderer
    .create(
      <Card
        url='https://static1.comicvine.com/uploads/scale_super/11118/111187046/5126812-geralt_ghoul_slayer_by_frostedflakes62-d9xd93e.jpg'
        title='Geralt Of Rivia'
        subtitle='The Blaviken Butcher'
      />
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
