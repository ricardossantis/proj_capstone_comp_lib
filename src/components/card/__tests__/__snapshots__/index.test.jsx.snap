// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderer 1`] = `
<div
  className="sc-bdnylx bBZxjW"
>
  <figure
    className="sc-gtssRu liqkNM"
  >
    <img
      className="sc-dlnjPT eJufFV"
      src="https://static1.comicvine.com/uploads/scale_super/11118/111187046/5126812-geralt_ghoul_slayer_by_frostedflakes62-d9xd93e.jpg"
    />
  </figure>
  <div
    className="sc-hKFyIo hhUFrW"
  >
    <h1>
      Geralt Of Rivia
    </h1>
    <h3>
      The Blaviken Butcher
    </h3>
  </div>
</div>
`;
